src/pci.rs

```diff
@@ -80,31 +80,36 @@ fn get_device_details(bus: u8, device: u8, func: u8) -> (u16, u16) {
 }
 
 pub fn print_bus() {
-    for device in 0..MAX_DEVICES {
-        let (vendor_id, device_id) = get_device_details(0, device, 0);
-        if vendor_id == INVALID_VENDOR_ID {
-            continue;
+    for bus in 0..MAX_BUSES {
```

Comment:
This will really harm boot time on CH where this only one bus (per segment.) Perhaps we could get the max bus from the appropriate ACPI table (yeh, I know currently we don't know about ACPI in this code but this is is stored in a static table.)

Reply:
Thank you for your review. This change was made during porting to aarch64, it seems to be unnecessary. Let's just close this PR.